ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -288,12 +308,14 @@ def start_learning(self):
         self._initialize_trainers(trainer_config)
         for _, t in self.trainers.items():
             self.logger.info(t)
-        global_step = 0  # This is only for saving the model
         curr_info = self._reset_env()
         if self.train_model:
             for brain_name, trainer in self.trainers.items():
                 trainer.write_tensorboard_text('Hyperparameters',
                                                trainer.parameters)
+            if sys.platform == 'win32':
```

Comment:
I am a bit confused here, is `sys.platform` really equal to `win32` even when the underlying OS is 64 bit windows?

Reply:
Yes even if os is 64 bit, sys.platform is win32, see https://stackoverflow.com/questions/2144748/is-it-safe-to-use-sys-platform-win32-check-on-64-bit-python.

Reply:
👍 Thinking a little bit more, let us make this `sys.platform.startswith('win')`, so that we defend against future versions of python where they make a separate `win64` string.

Reply:
Fixed